code/modules/admin/verbs/pray.dm

```diff
@@ -17,21 +17,22 @@
 		liaison = 1
 
 	if(liaison)
-		msg = SPAN_STAFF_IC("<b><font color=purple>LIAISON: </font>[key_name(src, 1)] [CC_MARK(src)] [ADMIN_PP(src)] [ADMIN_VV(src)] [ADMIN_SM(src)] [ADMIN_JMP_USER(src)] [ADMIN_SC(src)]:</b> [msg]")
+		msg = "<b><big>[SPAN_STAFF_IC("<font color=purple>LIAISON:</font>")][key_name(src, 1)] [CC_MARK(src)] [ADMIN_PP(src)] [ADMIN_VV(src)] [ADMIN_SM(src)] [ADMIN_JMP_USER(src)] [ADMIN_SC(src)]: [msg]</big></b>"
 	else
-		msg = SPAN_STAFF_IC("<b><font color=purple>PRAY: </font>[key_name(src, 1)] [CC_MARK(src)] [ADMIN_PP(src)] [ADMIN_VV(src)] [ADMIN_SM(src)] [ADMIN_JMP_USER(src)] [ADMIN_SC(src)]:</b> [msg]")
+		msg = "<b><big>[SPAN_STAFF_IC("<font color=purple>PRAY: </font>")][key_name(src, 1)] [CC_MARK(src)] [ADMIN_PP(src)] [ADMIN_VV(src)] [ADMIN_SM(src)] [ADMIN_JMP_USER(src)] [ADMIN_SC(src)]: [msg]</big></b>"
 	log_admin(msg)
 	for(var/client/C in GLOB.admins)
 		if(AHOLD_IS_MOD(C.admin_holder) && C.prefs.toggles_chat & CHAT_PRAYER)
 			to_chat(C, msg)
+			C << 'sound/machines/terminal_alert.ogg'
```

Comment:
does that work without the sound() ?

Reply:
Yes, it does. The USCM message does the same thing below and it also plays the sound.